proto/proto-backplane-grpc/src/main/proto/deephaven/proto/object.proto

```diff
@@ -12,7 +12,67 @@ option go_package = "github.com/deephaven/deephaven-core/go/internal/proto/objec
 import "deephaven/proto/ticket.proto";
 
 service ObjectService {
-    rpc FetchObject(FetchObjectRequest) returns (FetchObjectResponse) {}
+
+    /*
+     * Fetches a server-side object as a binary payload and assorted other tickets pointing at
+     * other server-side objects that may need to be read to properly use this payload. The binary
+     * format is implementation specific, but the implementation should be specified by the "type"
+     * identifier in the typed ticket.
+     *
+     * Deprecated in favor of MessageStream, which is able to handle the same content.
+     */
+    rpc FetchObject(FetchObjectRequest) returns (FetchObjectResponse) {
+        option deprecated = true;
+    }
+
+    /*
+     * Provides a generic stream feature for Deephaven instances to use to add arbitrary functionality.
+     * This API lets a client open a stream to a particular object on the server, to be mediated by
+     * a server side plugin. In theory this could effectively be used to "tunnel" a custom gRPC call,
+     * but in practice there are a few deliberate shortcomings that still make this possible, but not
+     * trivial.
+     *
+     * The first message sent to the server is expected to have a ConnectRequest, indicating which
+     * export ticket to connect to. It is an error for the server to not have a plugin defined for that
+     * object type.
+     *
+     * Subsequent requests to the server, and all responses to the client, will be a payload of bytes
+     * and an arbitrary number of tickets, referring to server side objects that one side wishes to
+     * reference in its payload.
```

Comment:
I think we should clarify "first message of a stream", "subsequent requests to the stream", etc.